src/Bitcoin/CashAddress.h

```diff
@@ -53,4 +56,23 @@ inline bool operator==(const CashAddress& lhs, const CashAddress& rhs) {
     return lhs.bytes == rhs.bytes;
 }
 
+class BitcoinCashAddress : public CashAddress {
+  public:
+    /// Initializes an address with a string representation.
+    explicit BitcoinCashAddress(const std::string& string)
+        : CashAddress("bitcoincash", string) {}
```

Comment:
can we add "bitcoincash" as a class static variable? please refer to `Zilliqa/Address.{h, cpp}` example